custom_report_label/views/custom_report_label_views.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding='UTF-8' ?>
+<odoo>
+    <record id="view_custom_report_label_form" model="ir.ui.view">
+        <field name="name">custom.report.label.form</field>
+        <field name="model">custom.report.label</field>
+        <field name="arch" type="xml">
+            <form string="Custom Report Label">
+                <sheet>
+                    <group>
+                        <field name="name" />
+                        <field name="model_id" />
+                        <field name="template_id" />
+                        <field name="existing_label" />
+                        <field name="custom_label" />
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id="view_custom_report_label_tree" model="ir.ui.view">
+        <field name="name">custom.report.label.tree</field>
+        <field name="model">custom.report.label</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+                <field name="model_id" />
+                <field name="template_id" />
```

Comment:
```suggestion
                <field name="template_id" />
                <field name="existing_label" />
                <field name="custom_label" />
```
I thought it would be better to have it in the list view as well.

Reply:
It seems good to do that.